spec/views/curation_concerns/file_sets/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe 'curation_concerns/file_sets/edit.html.erb', type: :view do
+  let(:user)    { create(:aic_user) }
+  let(:file_set) do
+    stub_model(FileSet, id: '123',
+                        depositor: user,
+                        resource_type: ['Dataset'])
+  end
+
+  let(:form) do
+    view.simple_form_for(file_set, url: '/update') do |fs_form|
+      return fs_form
+    end
+
+  end
+
+  before do
+    allow(controller).to receive(:current_user).and_return(user)
+
+    allow(AICUser).to receive(:find_by_nick).with(user).and_return(user)
+
+    view.lookup_context.prefixes.push 'curation_concerns/base'
+    render template: 'curation_concerns/file_sets/edit.html.erb', locals: { curation_concern: file_set }
```

Comment:
I can't really work through how to stub the file set such that the form thinks it has what it needs, and I can just check that it renders with the all of the UI elements it's given (including the wrapper). I looked at sufia's fileset form view tests for ideas. The above gives me a Blacklight record not found error. Explicitly trying to pass the form stub an array with main app as a double and the file_set like this ``` allow(form).to receive([main_app, file_set]) ``` gives me a different error, a blacklight one.